lib/formatters.js

```diff
@@ -43,6 +45,14 @@ const tags = (config, event) => {
         });
     }
 
+    if (values && Array.isArray(config.fieldTags)) {
```

Comment:
If a value is added to the tags, should it also be removed from fields? I guess duplication is not really a problem, just soliciting thoughts.

Reply:
Oh, that's true. I hadn't thought about that. I feel like tags can do all the things that fields can do, right? (just not vice-versa)

Reply:
Good discussion here on fields versus tags:
https://docs.influxdata.com/influxdb/v1.5/concepts/key_concepts/#discussion

Long story short: Fields are aggregate-able (sum, mean, min/max, etc). Tags are indexed; thus all queries should be performed against tags. So not any issue with saving fields twice, other than duplicate data.

Reply:
That's interesting (and good to know). Given this, I think it makes sense to keep it this way so that users can use the data however they want.